rocky/rocky/views/organization_member_edit.py

```diff
@@ -26,9 +26,11 @@ def test_func(self):
 
     def get_form(self):
         form = super().get_form()
-        if (self.object.user == self.request.user) or self.request.user.is_superuser:
+        if self.object.user.is_superuser or self.object.user.groups.filter(name=GROUP_ADMIN).exists():
             # There could be a case where you block yourself out of the system
             form.fields["status"].disabled = True
+        if self.object.user.groups.filter(name=GROUP_CLIENT).exists():
```

Comment:
This does two database queries to check if the user is in a specific group. We should just fetch the groups with self.request.user.groups.all() and then check if the specific group is in the list.